lib/brains/management/commands/braintree_webhook.py

```diff
@@ -0,0 +1,140 @@
+import base64
+import os
+from datetime import datetime, timedelta
+from optparse import make_option
+
+from django.core.management.base import BaseCommand, CommandError
+
+import braintree
+import requests
+from braintree.util.crypto import Crypto
+
+from lib.brains.models import BraintreeSubscription
+from lib.brains.tests.samples import webhooks
+from payments_config import products
+from solitude.logger import getLogger
+
+
+log = getLogger('s.brains.management')
+
+# These environment variables are populated by Docker. This script will not
```

Comment:
won't this code run even when this command doesn't run because of how Django imports it to discover the command? I think you should move this to the ``handle()`` function so that it only runs when the command runs.